application/templates/_shared/_phase_banner.html

```diff
@@ -1,19 +1,14 @@
 <div class="banner-container-outer">
   <div class="banner-container">
-    {% if config.SURVEY_ENABLED %}
```

Comment:
I think the differences here were a deliberate design decision... when the survey is active we wanted the beta banner to be more prominent I think, @frankieroberto ?

That said, I think this looks good all the same, so I'm easy either way.